src/vs/workbench/browser/parts/views/treeView.ts

```diff
@@ -1044,10 +1045,19 @@ class TreeRenderer extends Disposable implements ITreeRenderer<ITreeItem, FuzzyS
 		@IContextKeyService private readonly contextKeyService: IContextKeyService,
 	) {
 		super();
-		this._hoverDelegate = {
+		this._entryHoverDelegate = {
 			showHover: (options: IHoverDelegateOptions) => this.hoverService.showHover(options),
 			delay: <number>this.configurationService.getValue('workbench.hover.delay')
 		};
+
+		this._actionHoverDelegate = {
+			placement: 'element',
+			showHover: (options: IHoverDelegateOptions) => {
+				options.hoverPosition = HoverPosition.ABOVE;
```

Comment:
Can we use below instead so that we mimic OS tooltips better?

Reply:
The reason I chose ABOVE is because while native is below, its also slightly offset to account for the mouse position. Our hover service does seem to do this. This is why I went with above and with a beak.

Reply:
If you still prefer BOTTOM, we should improve the service to avoid the cursor.

Reply:
Oh, I forgot that the service does do this if you use mouse placement, but not if you choose element placement. Choosing mouse placement, the hover does not show for me, but I did not debug yet

Reply:
I would still put it below. We show the tree item hovers below, it would be inconsistent to show the action hovers above.